packages/feedback-service/src/resolver.ts

```diff
@@ -16,7 +17,26 @@ export const FeedbackResolver = {
     },
     getFeedbackBy(root: any,  args : any, ctx: any){
       return Feedback.find(args.input).exec();
-    }
+    },
+    getAllJiraIssues(root: any,  args : any, ctx: any) {
+    const jira = new JiraApi({
+    protocol: 'https',
+    host: process.env.JIRA_HOST,
+    username: process.env.JIRA_USERNAME,
+    password: process.env.JIRA_PASSWORD,
+    apiVersion: '2',
+    strictSSL: false
+   });
+   return  jira.getIssuesForBoard("3844")
```

Comment:
Just noticed this. The Board ID should not be hard coded. It should also be stored in an env variable.